### PR TITLE
add duplicate filter for ims

### PR DESCRIPTION
**Pull request by SteffenHeu**

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/builders/WizardBatchBuilder.java [hunk 1]
```diff
@@ -223,7 +225,14 @@ protected static void makeAndAddDuplicateRowFilterStep(final BatchQueue q,
     // going back into scans so rather use scan mz tol
     param.setParameter(DuplicateFilterParameters.mzDifferenceMax, mzTol);
     param.setParameter(DuplicateFilterParameters.rtDifferenceMax, rtTol);
+//    param.setParameter(DuplicateFilterParameters.mobilityDifferenceMax,
+//        mobilityType != MobilityType.NONE);
+//    param.getParameter(DuplicateFilterParameters.mobilityDifferenceMax).getEmbeddedParameter()
+//        .setValue(new MobilityTolerance(mobilityType == MobilityType.TIMS ? 0.008f : 1f));
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/builders/WizardBatchBuilder.java:231]: remove

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/BatchWizardTab.java [hunk 2]
```diff
@@ -408,7 +408,7 @@ public void createBatch() {
     }
 
     BatchModeParameters batchModeParameters = (BatchModeParameters) MZmineCore.getConfiguration()
-        .getModuleParameters(BatchModeModule.class);
+        .getModuleParameters(BatchModeModule.class).cloneParameterSet();
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/BatchWizardTab.java:411]: if we clone here we do not have to clone when calling runMZmineModule... with the parmaeters.
Can you remove the second call

**Review by robinschmid — DISMISSED** [review]
only some small comments. looks good otherwise

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/BatchWizardTab.java (hunk 2 above)
Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/modules/tools/batchwizard/BatchWizardTab.java:411]: that was just an attempt to fix the optional parameter not set correctly. will revert.

**Pull request merged**